client/src/components/shop/ShopBanner.js

```diff
@@ -0,0 +1,81 @@
+import React from "react";
+import Button from "@material-ui/core/Button";
+import CssBaseline from "@material-ui/core/CssBaseline";
+import Paper from "@material-ui/core/Paper";
+import Grid from "@material-ui/core/Grid";
+import Typography from "@material-ui/core/Typography";
+import { makeStyles } from "@material-ui/core/styles";
+
+const useStyles = makeStyles(theme => ({
+  image: {
+    backgroundRepeat: "no-repeat",
+    backgroundSize: "cover",
+    backgroundPosition: "center"
+  },
+  paper: {
+    margin: theme.spacing(8, 4),
+    display: "flex",
+    flexDirection: "column",
+    alignItems: "center",
+    height: "100vh",
+    justifyContent: "space-around"
+  },
+  editCoverBtn: {
+    margin: theme.spacing(2)
+  }
+}));
+
+export default function ShopBanner(props) {
+  const classes = useStyles();
+  return (
+    <Grid container component="main">
+      <CssBaseline />
```

Comment:
Why is CssBaseline here?

Reply:
I think because I copied code from a template. Will look into it.